src/sass/pages/_introduction.scss

```
@@ -0,0 +1,32 @@
+.main-introduction{
+    max-width: 1200px;
+    width: 100%;
+    padding-left: 15px;
+    padding-right: 15px;
+    margin: 0 auto;
+    .introduction-heading{
+        .section-header{
+            h2{
+                font-size: 2.4rem;
+            }
+        }
+    }
+    .introduction-content{
+        margin-top: 30px;
+        .content-img{
+            img{
+                width: 100%;
+            }
+        }
+        .content-text{
+            h2{
+                font-size: 2.4rem;
+                margin-bottom: 16px;
+            }
+            p{
+                font-size: 1.4rem;
+                color: $textColor;
+            }
+        }
+    }
+}
```

Bổ sung endline và chỉnh lại khoảng cách giữa selector với brackets
```suggestion
            p {
                font-size: 1.4rem;
                color: $textColor;
            }
        }
    }
}
```